randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -657,6 +660,8 @@ Extra - asset_id: collision_camera_006
   * Extra - right_shield_def: None
   > Door to Teleport to Dairon (Lower)
       Trivial
+  > Door to EMMI Zone First Entrance
+      Speed Booster
```

Comment:
This needs to have door lock rando disabled, and is thus pointless to have. Where are you going to charge the spark if you can't use any of the doors.